src/pages/home/ReportScreen.js

```diff
@@ -50,8 +56,8 @@ class ReportScreen extends React.Component {
      * @returns {Number}
      */
     getReportID() {
-        const params = this.props.route.params;
-        return Number.parseInt(params.reportID, 10);
+        const id = this.props.route.params.reportID || this.props.reportID;
```

Comment:
Currently during the first moments of this screen's life, routing would not provide `reportID` so we fall back to props.

Reply:
Nice thinking! Good compromise for now for sure.